### PR TITLE
Switches from deprecated webserver to Spring. Clean-up.

### PR DESCRIPTION
**Pull request by joeldudley**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-    ext.corda_release_version = '3.3-corda'
-    ext.corda_gradle_plugins_version = '3.2.1'
-    ext.kotlin_version = '1.1.60'
-    ext.junit_version = '4.12'
-    ext.quasar_version = '0.7.9'
+    ext {
+        corda_release_group = 'net.corda'
+        corda_release_version = '3.3-corda'
+        corda_gradle_plugins_version = '3.2.1'
+        kotlin_version = '1.1.60'
```
Comment (Clintonio) [inline build.gradle:6]: We should upgrade to Kotlin 1.2.70.

File: build.gradle [hunk 2]
```diff
@@ -122,11 +86,9 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
             address("localhost:10006")
             adminAddress("localhost:10046")
         }
-        webPort 10007
         cordapps = [
```
Comment (Clintonio) [inline build.gradle:101]: It's a shame the new `cordapp` block isn't available in 3.x.

**Review by Clintonio — CHANGES_REQUESTED** [review]
One minor thing.

File: build.gradle (hunk 1 above)
Comment (joeldudley) [reply build.gradle:6]: Agreed to leave as 1.1.60 for now.

File: build.gradle (hunk 2 above)
Comment (joeldudley) [reply build.gradle:101]: Yep

**Review by Clintonio — APPROVED** [review]

**Pull request merged**